### ISSUE TITLE
[Memcache] Setup memcache cluster

### ISSUE THREAD

**Issue by Shikkic** [gophr-pm/gophr#194]
For some of our future caching needs we'll want to setup a memcache cluster as part of our stack.

This doesn't take priority yet, but once we get into an optimization phase we'll want to be able to do a lot of caching resources.

**Comment by skeswa** [comment]
This has lower priority in the grand scheme of things.

**Comment by Shikkic** [comment]
Yep, I agree as well. Just writing this issue out so it can eventually get worked into the road map.
